### PR TITLE
feat(deletePodsAndServices):only delete master service

### PR DESCRIPTION
**Pull request by leileiwan**
for issue https://github.com/kubeflow/pytorch-operator/issues/232

### CODE REVIEW

File: pkg/controller.v1/pytorch/job.go [hunk 1]
```diff
@@ -148,7 +149,7 @@ func (pc *PyTorchController) updatePyTorchJob(old, cur interface{}) {
 	}
 }
 
-func (pc *PyTorchController) deletePodsAndServices(job *pyv1.PyTorchJob, pods []*v1.Pod) error {
+func (pc *PyTorchController) deletePodsAndServices(job *pyv1.PyTorchJob, pods []*v1.Pod,services []*v1.Service) error {
```
Comment (gaocegege) [inline pkg/controller.v1/pytorch/job.go]: I think we should have some comments to illustrate the behavior of the function.

Comment (leileiwan) [reply pkg/controller.v1/pytorch/job.go]: How about to use “delete all the pods and master servics”

Comment (gaocegege) [reply pkg/controller.v1/pytorch/job.go]: SGTM

Comment (gaocegege) [reply pkg/controller.v1/pytorch/job.go]:
```suggestion
// deletePodsAndServices deletes all the pods and master service.
func (pc *PyTorchController) deletePodsAndServices(job *pyv1.PyTorchJob, pods []*v1.Pod,services []*v1.Service) error {
```

Comment (gaocegege) [reply pkg/controller.v1/pytorch/job.go]: LGTM. Keeping the function name in the comment is sort of golang convention.

Comment (leileiwan) [reply pkg/controller.v1/pytorch/job.go]: Great

**Pull request merged**